frontend/src/tests/pages/Courses/CoursesCreatePage.test.js

```diff
@@ -91,7 +112,71 @@ describe("CoursesCreatePage tests", () => {
 
     // expect(quarterField).toHaveValue("20124");
     //expect(setQuarter).toBeCalledWith("20124"); //need this and axiosMock below?
```

Comment:
Even though this was in the original, pleas remove commented out code when you find it.
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html